commit c09440f7dcb304002dfced8c0fea289eb25f2da0
Author: Sabrina Dubroca <sd@queasysnail.net>
Date:   Fri Mar 11 18:07:33 2016 +0100

    macsec: introduce IEEE 802.1AE driver
    
    This is an implementation of MACsec/IEEE 802.1AE.  This driver
    provides authentication and encryption of traffic in a LAN, typically
    with GCM-AES-128, and optional replay protection.
    
    http://standards.ieee.org/getieee802/download/802.1AE-2006.pdf
    
    Signed-off-by: Sabrina Dubroca <sd@queasysnail.net>
    Reviewed-by: Hannes Frederic Sowa <hannes@stressinduktion.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/drivers/net/Makefile b/drivers/net/Makefile
index 900b0c5320bb..1aa7cb845663 100644
--- a/drivers/net/Makefile
+++ b/drivers/net/Makefile
@@ -10,6 +10,7 @@ obj-$(CONFIG_IPVLAN) += ipvlan/
 obj-$(CONFIG_DUMMY) += dummy.o
 obj-$(CONFIG_EQUALIZER) += eql.o
 obj-$(CONFIG_IFB) += ifb.o
+obj-$(CONFIG_MACSEC) += macsec.o
 obj-$(CONFIG_MACVLAN) += macvlan.o
 obj-$(CONFIG_MACVTAP) += macvtap.o
 obj-$(CONFIG_MII) += mii.o